packages/time-series/lib/commands/MGET.ts

```diff
@@ -18,14 +19,47 @@ export function pushFilterArgument(args: CommandArguments, filter: RedisVariadic
   return pushVariadicArguments(args, filter);
 }
 
+export type MGetRawReply2 = Array<[
```

Comment:
use `ArrayReply` instead of `Array`

Reply:
done and BlobStringReply instead of string and UnwrapReply in transformReply

Reply:
though had some issues with typing in the transformReply, needs to be validated.

Reply:
and same in RANGE/MRANGE/_WITHLABELS/